tests/drivers/uart/uart_basic_api/testcase.yaml

```diff
@@ -7,6 +7,13 @@ tests:
     harness: keyboard
     integration_platforms:
       - mps2_an385
+  drivers.uart.basic_api.wide:
+    extra_configs:
+      - CONFIG_UART_WIDE_DATA=y
+    tags: drivers uart
+    filter: ( CONFIG_UART_CONSOLE and CONFIG_SERIAL_SUPPORT_WIDE_DATA )
```

Comment:
I guess fixture is still required here (same as async version)
```
    harness: ztest
    harness_config:
      fixture: gpio_loopback
    depends_on: gpio
```

Reply:
Well, there's nothing to loopback here, it's just setting and getting configuration.

Reply:
Right, my bad. Approved for now.